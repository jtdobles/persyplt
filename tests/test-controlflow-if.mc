void main(){
	int a = 0;

	if (a > 1) {
		prints('a is greatr than 1');
	} else {
		prints('a less than or equal to 1');
	}
}