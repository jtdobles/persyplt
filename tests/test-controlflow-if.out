'a less than or equal to 1'
